scripts/buildhost/linux-arm64.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+BUILD_REQUEST=$1
+OUTPUTFILE=$2
+LOGFILE=$3
+BUCKET=$4
+NO_SIGN=$5
+echo "linux-arm64 $1 $2 $3 $4 $5"
+
+if [ "${BUILD_REQUEST}" = "" ]; then
+    echo "Missing BUILD_REQUEST argument"
+    exit 1
+fi
+
+SCRIPTPATH=$(pwd)
+TMPPATH=$(dirname ${BUILD_REQUEST})
+pushd .
+cd ${TMPPATH}
+
+AWS_REGION="us-west-2"
+AWS_LINUX_AMI="ami-0c579621aaac8bade"
+AWS_INSTANCE_TYPE="a1.2xlarge"
+INSTANCE_NUMBER=$RANDOM
+
+set +e
+
+${SCRIPTPATH}/start_ec2_instance.sh ${AWS_REGION} ${AWS_LINUX_AMI} ${AWS_INSTANCE_TYPE}
+if [ "$?" != "0" ]; then
```

Comment:
shouldn't shutdown_ec2_instance be called here as linux-arm.sh does?

Reply:
yes; I'll handle this on subsequent PR.